.travis.yml

```diff
@@ -7,7 +7,6 @@ arch:
 - arm64
```

Comment:
We're missing tests for the new envs.

Reply:
<img width="883" alt="CleanShot 2023-08-07 at 09 40 39@2x" src="https://github.com/utopia-php/system/assets/19310830/73c7906e-24b7-41a3-8b81-237697621ba6">

Seems like travis only support arm64, not other ones.